contributingGuides/CONTRIBUTING.md

```diff
@@ -114,6 +114,13 @@ Additionally if you want to discuss an idea with the open source community witho
 4. After you reproduce the issue, complete the [proposal template here](./PROPOSAL_TEMPLATE.md) and post it as a comment in the corresponding GitHub issue (linked in the Upwork job).
     - Note: Before submitting a proposal on an issue, be sure to read any other existing proposals. Any new proposal should be substantively different from existing proposals.
 5. Refrain from leaving additional comments until someone from the Contributor-Plus team and / or someone from Expensify provides feedback on your proposal (do not create a pull request yet).
+    - Do not leave more than one proposal.
+    - Do not make extensive changes to your current proposal until after it has been reviewed.
+    - If you want to make an entirely new proposal or update an existing proposal please go back and edit your original proposal and leave a comment like this:
```

Comment:
```suggestion
    - If you want to make an entirely new proposal or update an existing proposal, please go back and edit your original proposal, then post a new comment to the issue in this format to alert everyone that it has been updated:
```
This adds a little bit of information about _why_ we want the new comment

Reply:
Yep good clarification. Thanks.